uptimerobot/resource_uptimerobot_monitor_test.go

```diff
@@ -116,6 +116,50 @@ func TestUptimeRobotDataResourceMonitor_custom_port_monitor(t *testing.T) {
 	})
 }
 
+func TestUptimeRobotDataResourceMonitorAlertContact(t *testing.T) {
+	var FriendlyName = "TF Test: http monitor"
+	var Type = "http"
+	var URL = "https://google.com"
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckMonitorDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: fmt.Sprintf(`
+				resource "uptimerobot_alert_contact" "test" {
+					friendly_name = "#infrastructure"
+					type = "slack"
+					value = "https://slack.com/services/xxxx"
+				}
+				resource "uptimerobot_monitor" "test" {
+					friendly_name = "%s"
+					type          = "%s"
+					url           = "%s"
+					alert_contact {
+						id         = "${uptimerobot_alert_contact.test.id}"
+						threshold  = 0
+						recurrence = 0
+					}
+				}
+				`, FriendlyName, Type, URL),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("uptimerobot_monitor.test", "friendly_name", FriendlyName),
+					resource.TestCheckResourceAttr("uptimerobot_monitor.test", "type", Type),
+					resource.TestCheckResourceAttr("uptimerobot_monitor.test", "url", URL),
+					resource.TestCheckResourceAttr("uptimerobot_monitor.test", "alert_contact.#", "1"),
```

Comment:
If we add a couple more `resource.TestCheckResourceAttr` here we can make sure that TF doesn't try to update the resource every time

```suggestion
					resource.TestCheckResourceAttr("uptimerobot_monitor.test", "alert_contact.#", "1"),
					resource.TestCheckResourceAttrSet("uptimerobot_monitor.test", "alert_contact.0.id"),
					resource.TestCheckResourceAttr("uptimerobot_monitor.test", "alert_contact.0.threshold", "0"),
					resource.TestCheckResourceAttr("uptimerobot_monitor.test", "alert_contact.0.recurrence", "0"),
```